homeassistant/components/awattar/config_flow.py

```diff
@@ -0,0 +1,48 @@
+"""Config flow for aWATTar integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_COUNTRY
+from homeassistant.helpers.selector import CountrySelector, CountrySelectorConfig
+
+from .const import DOMAIN
+
+SUPPORTED_COUNTRIES = ["AT", "DE"]
+
+
+class AwattarFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Config flow for aWATTar integration."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+
+        await self.async_set_unique_id(DOMAIN)
+        self._abort_if_unique_id_configured()
```

Comment:
This has the side-effect of only be able to configure a single config entry, even though it is possible that a user wants to see the data for both countries.

Reply:
This is not supported to see the data of both countries. even this does not make sense, as you cant get energy from both countries at once.
If we would like to support this (which only makes sense to compare the values of the different countries) we need to put the country code in the entity-id.

Reply:
Hmm, I don't see a reason why we should forbid to track both countries, even though the use case is limited. But maybe @jpbede can give his opinion on that.

Reply:
But then we have to put the country code in the entity-id's. Or is there another option ?
Looks like we have to streamline all integrations providing this kind of data :)

I found the following integrations providing similar data (only the first two acting similar):

- https://www.home-assistant.io/integrations/energyzero/
- https://www.home-assistant.io/integrations/easyenergy/
- https://www.home-assistant.io/integrations/pvpc_hourly_pricing/
- https://www.home-assistant.io/integrations/tibber/

So maybe we should think about how to present energy market prices to the user in general ? 
Who is responsible for such (architectural) decisions ?

Reply:
Based on the background that you just have one energy contract, it would make sense to limit this to a single config entry imo.

So `self._abort_if_unique_id_configured()` can be removed, as HA handles this when the manifest option is set.